src/models/user-played-time-response.model.ts

```diff
@@ -0,0 +1,44 @@
+import type { GameConcept } from "./title-concept.model";
+import type { Media } from "./game-media.model";
+
+export interface GetUserPlayedTimeResponse {
+  /** The title ID of this game 
+      @example CUSA26431_00
+      */
+  titleId: string;
+
+  /** Default name for the title */
+  name: string;
+
+  /** Localized name for the title */
+  localizedName: string;
+
+  /** URL to game's console launch screen, from image.api.playstation.com */
+  imageUrl: string;
+
+  /** URL to localized game's console launch screen. Will default to imageUrl if not unique */
+  localizedImageUrl: string;
+
+  /** Type of game being played. Can be ps5_native_game, ps4_game or none */
+  category: string;
+
+  /** Service where this game was acquired from: EA Access, none(purchased/none_purchased, ps_plus, other, */
+  service: string;
+
+  /** Metadata about this game, including titleIds and localization for other regions, genres, name and id */
+  concept: GameConcept;
+
+  /** An object with audio, video, images properties that will contain arrays of objects with URL, format and type properties */
+  media: Media;
+
+  /** A Date object in ISO format */
+  firstPlayedDateTime: Date;
+
+  /** A Date object in ISO format */
+  lastPlayedDateTime: Date;
+
+  /**
+   * @example `"PT11H16M43S"`
+   */
+  playDuration: string;
```

Comment:
The example is good but we definitely need to elaborate on what this example means so our comment can serve as a reference.

Reply:
My bad, forgot to do that when I was drafting the PR. All playDuration strings follow this format:

`PT` + `{Number of hours, if any}H` + `{Number of minutes, if any}M` + `{Number of seconds, if any}S`

How do you suggest improving this comment?

Reply:
Don't know if I wasn't clear or if I just don't understand what strong typing is, so I'm just going to paste a few of the playtime durations you can get from this:
```
playDuration: 'PT1H14M10S'
playDuration: 'PT10H28M12S'
playDuration: 'PT1H52M23S'
playDuration: 'PT2M38S'
playDuration: 'PT7M30S'
playDuration: 'PT69H17M26S'
```

All of these are from the user `pandaswithbombs`, if you want to double check it.
Does this data fits the format that you suggested? If so, I'll just commit what you said.

Reply:
I have done some research on this value and it seems it's actually a technical standard: an ISO 8601 duration string. To be honest, this is the first time I've ever come across this.

After reading a bit more about the spec, we probably can't implement a strong type for this and will have to use `string`. However, we should document on the field explicitly that it's an ISO 8601 duration string.

ref: https://tc39.es/proposal-temporal/docs/duration.html